src/pq/connection.cr

```diff
@@ -304,7 +320,14 @@ module PQ
         i = params.find { |p| p[0] == 'i' }.not_nil![2..-1].to_i
         raise ConnectionError.new("SASL: scram server nonce does not start with client nonce") unless r.starts_with?(@client_nonce)
 
-        client_final_msg_without_proof = "c=biws,r=#{r}"
+        if @signature
+          # cd10b...Cws == base64 of "p=tls-server-end-point,,"
+          b64sig = Base64.strict_encode @signature.not_nil!
+          client_final_msg_without_proof = "c=cD10bHMtc2VydmVyLWVuZC1wb2ludCws#{b64sig},r=#{r}"
```

Comment:
Why not include encoding the static part (`p=tls-server-end-point,,`)? I think this makes the code much easier to read.
Performance impact should be completely negligible.